cloud/kubernetes/helm/yugabyte/templates/service.yaml

```diff
@@ -212,8 +224,13 @@ spec:
           - "--fs_data_dirs={{ template "yugabyte.fs_data_dirs" $storageInfo }}"
           - "--rpc_bind_addresses={{ template "yugabyte.server_address" . }}"
           - "--server_broadcast_addresses={{ template "yugabyte.server_address" . }}:7100"
+          {{ if $root.Values.isMultiAz }}
+          - "--master_addresses={{ $root.Values.masterAddresses}}"
+          - "--replication_factor={{ $root.Values.replicas.totalMasters }}"
```

Comment:
You would need the masters to come up with the correct placement info. See this step: https://docs.yugabyte.com/latest/deploy/public-clouds/aws/#create-yb-master1-s-config-file

There should be a corresponding thing done for the tservers also.

Reply:
Those flags are passed while creating the universe through YW.

Reply:
Got it!